docs_src/microservices/device/Ch-DeviceServices.md

```diff
@@ -176,6 +176,34 @@ Please refer to the general [Common Configuration documentation](../configuratio
     |---|---|---|
     |MaxEventSize|0|maximum event size in kilobytes sent to Core Data or MessageBus. 0 represents default to system max.|
 
+### URIs for Device Service Files
```

Comment:
Please update the `DevicesDir`, `ProfilesDir`, `ProvisionWatchersDir` descriptions in the `Device` tab to talk about URI and to see this section.